spec.html

```diff
@@ -109,18 +107,12 @@ <h1>Introduction</h1>
   <p>ECMAScript 2020, the 11<sup>th</sup> edition, introduces the `matchAll` method for Strings, to produce an iterator for all match objects generated by a global regular expression; `import()`, a syntax to asynchronously import Modules with a dynamic specifier; `BigInt`, a new number primitive for working with arbitrary precision integers; `Promise.allSettled`, a new Promise combinator that does not short-circuit; `globalThis`, a universal way to access the global `this` value; dedicated `export * as ns from 'module'` syntax for use within modules; increased standardization of `for-in` enumeration order; `import.meta`, a host-populated object available in Modules that may contain contextual information about the Module; as well as adding two new syntax features to improve working with &ldquo;nullish&rdquo; values (`null` or `undefined`): nullish coalescing, a value selection operator; and optional chaining, a property access and function invocation operator that short-circuits if the value to access/invoke is nullish.</p>
   <p>ECMAScript 2021, the 12<sup>th</sup> edition, introduces the `replaceAll` method for Strings; `Promise.any`, a Promise combinator that short-circuits when an input value is fulfilled; `AggregateError`, a new Error type to represent multiple errors at once; logical assignment operators (`??=`, `&&=`, `||=`); `WeakRef`, for referring to a target object without preserving it from garbage collection, and `FinalizationRegistry`, to manage registration and unregistration of cleanup operations performed when target objects are garbage collected; separators for numeric literals (`1_000`); and `Array.prototype.sort` was made more precise, reducing the amount of cases that result in an implementation-defined sort order.</p>
   <p>Dozens of individuals representing many organizations have made very significant contributions within Ecma TC39 to the development of this edition and to the prior editions. In addition, a vibrant community has emerged supporting TC39's ECMAScript efforts. This community has reviewed numerous drafts, filed thousands of bug reports, performed implementation experiments, contributed test suites, and educated the world-wide developer community about ECMAScript. Unfortunately, it is impossible to identify and acknowledge every person and organization who has contributed to this effort.</p>
-  <p>
-    Allen Wirfs-Brock<br>
-    ECMA-262, Project Editor, 6<sup>th</sup> Edition
-  </p>
-  <p>
-    Brian Terlson<br>
-    ECMA-262, Project Editor, 7<sup>th</sup> through 10<sup>th</sup> Editions
-  </p>
-  <p>
-    Jordan Harband<br>
-    ECMA-262, Project Editor, 10<sup>th</sup> through 12<sup>th</sup> Editions
-  </p>
+  <p>Allen Wirfs-Brock<br>
+    ECMA-262, Project Editor, 6<sup>th</sup> Edition</p>
```

Comment:
oof, this is really gross - can we not make the `<p>` indentation match the `</p>`?

Reply:
> oof, this is really gross - can we not make the `<p>` indentation match the `</p>`?

I'm not sure what you're suggesting; can you write it out?

Reply:
Basically the way it was before the change. `<p>`, a newline, the indented content, a newline, and `</p>`, just like any OTBS style would do with braces.

Reply:
Every other paragraph in the spec has the text on the same line as the `<p>`. I am reluctant to relax that rule and I don't really want to special-case these.

Reply:
Yes they do; there's something like a dozen cases that have newlines inside of the `<p>` and also have the text on the same line as the tags.

I'm OK adding a rule which says that if there are multiple lines in the `<p>` then it goes on a different line than the opening tag, but that's _not_ the rule we currently follow.

Reply:
Pushed a commit using the proposed rule. It will fail CI unless/until https://github.com/tc39/ecmarkup/pull/369 is included in this PR, but you can see what it looks like, at least.

Reply:
thanks, that looks SO much better.